doc/rst/language/spec/arrays.rst

```diff
@@ -625,6 +632,21 @@ case, each element in the array is assigned this value.
       77 77 77 77
       33 33 33 33
 
+
+Array Comparison
+----------------
+
+Arrays are promoted, so the result of the equality operators is
```

Comment:
I know this is something you moved, but I'd say "Because `==` on arrays is promoted, ..." since I'd view it as the operation on the array being promoted, not the arrays themselves

Reply:
Changed it to:

With arrays, the equality operator (i.e. ==) is promoted, so the result is an
array of booleans.  To get a single result use the ``equals`` method instead.